app/components/Analyst/Project/Announcements/AnnouncementDeleteModal.tsx

```diff
@@ -101,34 +100,9 @@ const DeleteModal = ({
         ) : (
           <p>Are you sure you want to delete this announcement?</p>
         )}
-        <ModalButtons>
-          {isMultiProject && (
-            <Modal.Close>
-              <Button
-                onClick={handleDeleteAll}
-                data-testid="delete-from-all-btn"
-              >
-                Delete from all projects
-              </Button>
-            </Modal.Close>
-          )}
-          <Modal.Close>
-            <Button
-              onClick={handleDeleteOne}
-              data-testid="delete-from-this-btn"
-            >
-              {isMultiProject ? `Remove from this project` : `Yes, delete`}
-            </Button>
-          </Modal.Close>
-          <Modal.Close>
-            <Button variant="secondary">
-              {!isMultiProject && `No, `}Cancel
-            </Button>
-          </Modal.Close>
-        </ModalButtons>
-      </Modal.Content>
-    </StyledModal>
+      </p>
```

Comment:
You've got some extra `<p>` tags here that I think we can remove

Reply:
Yep, removing it